src/modules/pokemons/RoamingPokemonList.ts

```diff
@@ -136,6 +137,7 @@ RoamingPokemonList.add(Region.kalos, 0, new RoamingPokemon('Ash-Greninja', new T
 RoamingPokemonList.add(Region.alola, 0, new RoamingPokemon('Magearna', new GymBadgeRequirement(BadgeEnums.Champion_Stamp)));
 RoamingPokemonList.add(Region.alola, 0, new RoamingPokemon('Marshadow', new GymBadgeRequirement(BadgeEnums.Champion_Stamp)));
 RoamingPokemonList.add(Region.alola, 0, new RoamingPokemon('Zeraora', new GymBadgeRequirement(BadgeEnums.Champion_Stamp)));
+RoamingPokemonList.add(Region.alola, 0, new RoamingPokemon('Mimikyu (Busted)', new DefeatedSpecificPokemonRequirement(778, 5, true)));
```

Comment:
```suggestion
RoamingPokemonList.add(Region.alola, 0, new RoamingPokemon('Mimikyu (Busted)', new StatisticRequirement(['shinyPokemonDefeated', PokemonHelper.getPokemonByName('Mimikyu').id], 5, 'Defeat 5 shiny Mimikyu')));
```
We should not have [more] requirement duplicates. StatisticRequirement already handles this just fine.